src/fmu/dataio/load/load_standard_results.py

```diff
@@ -0,0 +1,101 @@
+from io import BytesIO
+from pathlib import Path
+from typing import Generic, TypeVar
+from uuid import UUID
+
+from pandas import DataFrame
+
+from fmu.dataio._models.fmu_results import enums
+from fmu.dataio.export._decorators import experimental
+from fmu.dataio.external_interfaces.schema_validation_interface import (
+    SchemaValidationInterface,
+)
+from fmu.dataio.external_interfaces.sumo_explorer_interface import SumoExplorerInterface
+
+T = TypeVar("T")
+
+
+class StandardResultsLoader(Generic[T]):
+    """The generic class for loaded standard results in fmu-dataio."""
+
+    def __init__(
+        self, case_id: UUID, ensemble_name: str, standard_result_name: str
+    ) -> None:
+        self._sumo_interface = SumoExplorerInterface(
+            case_id, ensemble_name, standard_result_name
+        )
+
+    def list_realizations(self) -> list[int]:
+        return self._sumo_interface.get_realization_ids()
+
+    def concatenate_realizations(self) -> T:
+        raise NotImplementedError
+
+    def save_realization(self, realization_id: int, folder_path: str) -> list[str]:
+        realization_data: list[tuple[DataFrame, dict]] = (
+            self._sumo_interface.get_realization_with_metadata(realization_id)
+        )
+
+        file_paths: list[str] = []
+        for data in realization_data:
+            data_frame: DataFrame = data[0]
+            metadata: dict = data[1]
```

Comment:
This could be done through unpacking directly 🙂 
```suggestion
        for data_frame, metadata in realization_data:
```